contrib/memory/org.nitlanguage.memory.txt

```diff
@@ -0,0 +1,16 @@
+Categories:Nit,Games
```

Comment:
Not really a remark, but I wonder what this file is for, is it some kind of Android metadata ?